bookwyrm/templates/lists/curate.html

```diff
@@ -2,49 +2,71 @@
 {% load i18n %}
 {% block panel %}
 
-<section class="content block">
-    <h2>{% trans "Pending Books" %}</h2>
-    <p><a href="{% url 'list' list.id %}">{% trans "Go to list" %}</a></p>
+<section class="block">
+    <div class="columns is-mobile is-multiline is-align-items-baseline">
+        <div class="column is-narrow">
+            <h2 class="title is-4">{% trans "Pending Books" %}</h2>
+        </div>
+
+        <p class="column is-narrow"><a href="{% url 'list' list.id %}">{% trans "Go to list" %}</a></p>
+    </div>
+
     {% if not pending.exists %}
-    <p>{% trans "You're all set!" %}</p>
+        <p>{% trans "You're all set!" %}</p>
     {% else %}
-    <table class="table is-striped">
-        <tr>
-            <th></th>
-            <th>{% trans "Book" %}</th>
-            <th>{% trans "Suggested by" %}</th>
-            <th></th>
-        </tr>
+
+    <dl>
```

Comment:
@joachimesque: I implemented the list I mentioned in the past. I think it works fine but I can replace it with a table if need be.